save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -615,10 +633,40 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
         }
     }
 
+    /**
+     * In some cases scripts and binaries can be uploaded to a git repository, so users won't be providing or uploading
+     * binaries. For this case we should open a window, so user will need to click a check box, so he will confirm that
+     * he understand what he is doing.
+     */
+    private fun submitWithValidation() {
+        setState {
+            isSubmitButtonPressed = true
+        }
+        when {
+            // url was not provided
+            gitUrlFromInputField.isNullOrBlank() && state.isFirstTypeUpload!! -> setState {
+                isErrorOpen = true
+                errorMessage = "Git Url with test suites in save format was not provided. It is required for processing."
+                errorLabel = "Git Url"
+            }
+            // no binaries were provided
+            state.files.isEmpty() -> setState {
+                confirmationType = ConfirmationType.NO_BINARY_CONFIRM
+                isConfirmWindowOpen = true
+                confirmLabel = "My label"
+                confirmMessage = "You have not provided any files related to your tested tool." +
+                        " If these files were uploaded to your repository - press OK, otherwise - please upload these files to save."
```

Comment:
```suggestion
                        " If these files were uploaded to your repository - press OK, otherwise - please upload these files using 'Upload files' button."
```
Maybe like this?